tests/playwright/utils/env.ts

```diff
@@ -1,48 +1,209 @@
-import { Page } from "@playwright/test";
-
-export interface EnvironmentConfig {
-  userPoolId: string;
-  clientId: string;
-  identityPoolId?: string;
-  apiUrl: string;
-  cognitoRegion?: string;
+import { expect, Page } from "@playwright/test";
+import { cognitoIdentityRoute } from "./consts";
+
+/**
+ * Authenticate using UI login
+ * @param page - Playwright page object
+ * @param username - User's email/username
+ * @param password - User's password
+ * @param expectedHeading - Expected heading text after successful login
+ * @param userType - 'ADMIN' or 'STATE' for environment variable prefixing
+ */
+export async function authenticateWithUI(
+  page: Page,
+  username: string,
+  password: string,
+  expectedHeading: string,
+  userType: "ADMIN" | "STATE"
+): Promise<void> {
+  await page.goto("/");
+
+  const emailInput = page.getByRole("textbox", { name: "email" });
+  const passwordInput = page.getByRole("textbox", { name: "password" });
+  const loginButton = page.getByRole("button", { name: "Log In with Cognito" });
+
+  await emailInput.fill(username);
+  await passwordInput.fill(password);
+  await loginButton.click();
+  await interceptAndStoreIdentityRequest(page, userType);
+
+  await page.waitForURL("/");
+  await expect(
+    page.getByRole("heading", { name: expectedHeading })
+  ).toBeVisible();
 }
 
-// Extract all environment configuration from env-config.js
-export async function getEnvironmentConfig(
+/**
+ * Intercept Cognito Identity requests to capture AWS credentials
+ * @param page - Playwright page object
+ * @param userType - 'ADMIN' or 'STATE' for environment variable prefixing
+ */
+export async function interceptAndStoreIdentityRequest(
+  page: Page,
+  userType: "ADMIN" | "STATE"
+): Promise<void> {
+  let credentialsCaptured = false;
+
+  const routeHandler = async (route: any) => {
+    const request = route.request();
+
+    try {
+      // Only intercept POST requests
+      if (request.method() === "POST") {
+        const response = await route.fetch();
+
+        // Check if this response contains credentials
+        try {
+          const responseBody = await response.json();
+
+          if (
+            responseBody.Credentials &&
+            responseBody.Credentials.AccessKeyId
+          ) {
+            process.env[`${userType}_AWS_ACCESS_KEY_ID`] =
+              responseBody.Credentials.AccessKeyId;
```

Comment:
When I log into MFP and inspect my local storage, I don't see an these values.  Shouldn't we be using the values the frontend uses?